frontend-react/src/pages/tos-sign/SuccessPage.tsx

```diff
@@ -0,0 +1,98 @@
+import React from "react";
+import { Helmet } from "react-helmet";
+
+import Title from "../../components/Title";
+
+import { AgreementBody } from "./TermsOfServiceForm";
+
+const Step = ({
+    heading,
+    label,
+}: {
+    number: number;
+    heading: string;
+    label: string;
+    complete: boolean;
+}) => {
+    return (
+        <li className="usa-process-list__item">
+            <h4 className="usa-process-list__heading">{heading}</h4>
+            <p className="margin-top-05">{label}</p>
+        </li>
+    );
+};
+
+function SuccessPage({ data }: { data: AgreementBody | null }) {
+    const steps = [
+        {
+            number: 1,
+            heading: "Account creation",
+            label: "The ReportStream team will create and configure your account.",
+            complete: false,
+        },
+        {
+            number: 2,
+            heading: "Log in to your account",
+            label: "The ReportStream team will reach out to you within one week with login credentials.",
+            complete: false,
+        },
+    ];
+    return data !== null ? (
+        <>
+            <Helmet>
+                <title>
+                    Sign the Terms of Service | {process.env.REACT_APP_TITLE}
+                </title>
+            </Helmet>
+            <div
+                data-testid="success-container"
+                className="tablet:grid-col-6 margin-x-auto"
+            >
+                <Title
+                    preTitle="Account registration"
+                    title={`You're almost there, ${data.firstName}!`}
+                />
+                <div className="usa-alert usa-alert--success">
+                    <div className="usa-alert__body">
+                        <h4 className="usa-alert__heading">
+                            Registration request submitted
+                        </h4>
+                        <p className="usa-alert__text">
+                            <strong>Name:</strong> {data.firstName}{" "}
+                            {data.lastName}
+                            <br />
+                            <strong>Email:</strong> {data.email}
+                            <br />
+                            <strong>State or territory:</strong>{" "}
+                            {data.territory.toUpperCase()}
+                            <br />
+                            <strong>Organization name:</strong>{" "}
+                            {data.organizationName}
+                            <br />
+                            <br />A copy of this confirmation has been sent to{" "}
+                            {data.email}. If you don't receive confirmation,
+                            check your SPAM folder for an email from
+                            reportstream@cdc.gov.
+                        </p>
```

Comment:
The above section's approach is fine for now, but the {" "} and<br/>s are usually an indicator that a more structured html/css approach might work better (e.g. Trusswork's grid is a nice concise way to do this).

If @drew-usds is ok with this section so we can get this PR merged, them I'm ok with it. We can optimize it for maintainability later.

Reply:
I've seen this a few places in the app, and I think I refactored some of them out effectively. I wonder if this content on the page is something that we could write as markdown once we get markdown support in the react app? Would love thoughts @drew-usds

Reply:
@kevinhaube @TomNUSDS Ah yeah this _should_ be the alert component, eh? As long as it works for now that's fine by me. We can always come back and refactor.